source/core/src/main/com/csse3200/game/components/player/PlayerStatsDisplay.java

```diff
@@ -94,75 +126,69 @@ private void addActors() {
         CharSequence healthText = String.format("Health: %d", health);
         healthLabel = new Label(healthText, skin, "small");
 
-        //Speed image
+        // Speed image
         float speedSideLength = 100f;
         speedImage = new Image(ServiceLocator.getResourceService().getAsset(SPEED_TEXTURE, Texture.class));
 
-        //Speed text
+        // Speed progress bar
         speedProgressBar = new ProgressBar(0f, 1.5f, 0.1f, false, skin);
         speedProgressBar.setWidth(200f);
         speedProgressBar.setAnimateDuration(2.0f);
-        /*
-        //Temporarily commented out in case design team prefers text instead of progress bar
-        float speedPercentage = entity.getComponent(PlayerActions.class).getCurrSpeedPercentage();
-        CharSequence speedText = String.format("Speed: %.1f%%", speedPercentage);
-        speedLabelText = new Label(speedText, skin, "small");
-         */
-
-        //Damage Progress bar
-        //Will need to check values
+
+        // Damage progress bar
         float damageSideLength = 50f;
-//        damageImage = new Image(
-//                ServiceLocator.getResourceService().getAsset("images/heart.png", Texture.class));
         damageImage = new Image(ServiceLocator.getResourceService().getAsset(DAMAGE_BUFF_TEXTURE, Texture.class));
         damageProgressBar = new ProgressBar(0f, 5.0f, 0.1f, false, skin);
         damageProgressBar.setWidth(200f);
         damageProgressBar.setAnimateDuration(2.0f);
 
-        //Weapon text, like the name of weapon
-        //entity.getComponent(WeaponComponent.class).getWeaponType();
-        pickaxeLabel = new Label("Pickaxe: 0", skin, "small");
-        shotgunLabel = new Label("Shotgun: 0", skin, "small");
-
+        // Weapon labels
+        swordLabel = new Label("Sword: 0", skin, "small");
```

Comment:
No need to have sword, shotgun and pistol separately, only need "gun" and "melee" or something like that.